README.md

```diff
@@ -711,16 +711,33 @@ meant to be able to change with it.
     easier understanding and reading of a test.
 
   * <a name="factories"></a>
-    Use [Factory Bot](https://github.com/thoughtbot/factory_bot) to create test
-    objects in integration tests. You should very rarely have to use
-    `ModelName.create` within an integration spec. Do **not** use fixtures as they
-    are not nearly as maintainable as factories.
+    Use [Factory Bot](https://github.com/thoughtbot/factory_bot) to
+    create test data in integration tests. You should very rarely
+    have to use `ModelName.create` within an integration spec. Do
+    **not** use fixtures as they are not nearly as maintainable as
+    factories.
     <sup>[[link](#factories)]</sup>
 
     ```ruby
-    subject(:article) { FactoryBot.create(:some_article) }
+    # bad
+    subject(:article) do
+      Article.create(
+        title: 'Piccolina',
+        author: 'John Archer',
+        published_at: '17 August 2172',
+        approved: true
+      )
+    end
+
+    # good
+    subject(:article) { FactoryBot.create(:article) }
     ```
 
+    *NOTE*: When talking about unit tests the best practice would be to
+    use neither fixtures nor factories. Put as much of your domain logic
+    in libraries that can be tested without needing complex, time
+    consuming setup with either factories or fixtures.
```

Comment:
I believe the classic definition of unit tests is when you test code without any dependencies (database, filesystem, cache etc.). Factories don’t necessarily violate that rule, e.g. `FactoryBot` provides a `#build` method that won’t try to save the object.

Reply:
I think it was meant that even `build_stubbed` can hit the DB as long as the factory is improperly defined e.g.
```ruby
after(:stub) { |model, _evaluator| model.something = create(:something) }
```

Also last time I measured a factory without any associations or traits, it took 1x for `Model.new`, 2x for `instance_double('Model')` and 10x for `build_stubbed(:model)`.
If one has a chance to define proper boundaries and write unit tests, I would be surprised to see factories involved.